### PR TITLE
Feat/colors

### PR DESCRIPTION
**Pull request by maycon-mello**
- Setup colors following styleguide
- Create theme
- Create ThemeProvider
- Create Storybook story

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (maycon-mello) [reply .vscode/settings.json]: Sure

File: packages/identity-wallet-mobile/package.json [hunk 2]
```diff
@@ -23,6 +23,8 @@
     "react": "^16.9.0",
     "react-native": "^0.60.5",
     "react-native-gesture-handler": "^1.3.0",
+    "react-native-paper": "^2.16.0",
```
[the remark this answers is not in the document]
Comment (maycon-mello) [reply packages/identity-wallet-mobile/package.json]: Right, will do it

File: packages/identity-wallet-mobile/.flowconfig [hunk 3]
```diff
@@ -42,6 +42,8 @@ esproposal.nullish_coalescing=enable
 module.file_ext=.js
```
[the remark this answers is not in the document]
Comment (maycon-mello) [reply packages/identity-wallet-mobile/.flowconfig]: react native core components are written using flowtype for prop validations, so react native apps usually follow the same structure to write react native components. Flow would not be used in the core packages and redux ducks, just on Prop validation for components. 

There is an example here: https://github.com/facebook/react-native/blob/master/Libraries/Modal/Modal.js#L77

do you agree with it?

**Comment by sk91** [comment]
Yep, sounds right

**Pull request merged**